libraries/DNSServer/src/DNSServer.h

```diff
@@ -1,5 +1,7 @@
 #ifndef DNSServer_h
 #define DNSServer_h
+
+#include <memory>
```

Comment:
future-proofing, I tried adding DNSServer/examples/DNSServer to host test script, but I see valgrind error in UDP code :shrug:
something to note

Reply:
Is there a line number to go with that?

Reply:
With this change:
```
--- a/libraries/DNSServer/examples/DNSServer/DNSServer.ino
+++ b/libraries/DNSServer/examples/DNSServer/DNSServer.ino
@@ -1,5 +1,5 @@
-#include <ESP8266WiFi.h>
 #include <DNSServer.h>
+#include <ESP8266WiFi.h>
 #include <ESP8266WebServer.h>
```

This happens:
```
C++ ...libraries/DNSServer/examples/DNSServer/DNSServer.ino.cpp.o
In file included from ./../../libraries/DNSServer/examples/DNSServer//DNSServer.ino:1,
                 from ...libraries/DNSServer/examples/DNSServer/DNSServer.ino.cpp:1:
../../libraries/DNSServer/src/DNSServer.h:113:10: error: ‘unique_ptr’ in namespace ‘std’ does not name a template type
  113 |     std::unique_ptr<DNSS_REQUESTER[]> _que;
      |          ^~~~~~~~~~
../../libraries/DNSServer/src/DNSServer.h:4:1: note: ‘std::unique_ptr’ is defined in header ‘<memory>’; did you forget to ‘#include <memory>’?
    3 | #include <WiFiUdp.h>
  +++ |+#include <memory>
    4 | 
```